.github/workflows/upgrade-containers.yml

```
@@ -0,0 +1,33 @@
+        uses: actions/checkout@v4
+
+      - name: Set up timestamp
+        id: get-timestamp
+        run: echo "timestamp=$(date +%Y-%m-%d-%H-%M-%S)" >> $GITHUB_OUTPUT
+
+      - name: Upgrade containers
+        run: |
+          find . -name 'Dockerfile' -exec sed -i "s/ARG CACHEBUST=.*/ARG CACHEBUST=${{ steps.get-timestamp.outputs.timestamp }}/" {} \;
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)